README.md

```diff
@@ -42,7 +42,7 @@ To keep this repo simple without board specific dependencies, we have a separate
 - STM32L4+
 - STM32H745
 - NXP RT1060
-- ESPRESSIF ESP32
+- ESPRESSIF ESP32 (Azure IoT Hub, Azure Device Update)
 - Linux
```

Comment:
Should this information be on Linux and the L475 bullet as well?

Reply:
Yes. Good catch! Thanks. Done.